packaging/installer/functions.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 # SPDX-License-Identifier: GPL-3.0-or-later
-# shellcheck disable=SC1091,SC1117,SC2002,SC2004,SC2034,SC2046,SC2059,SC2086,SC2129,SC2148,SC2154,SC2155,SC2162,SC2166,SC2181,SC2193
```

Comment:
Thanks! 🤣Let's actively remove these from user-facing shell scripts as we do other things. 👍